server/builtin/list.go

```diff
@@ -56,7 +56,7 @@ func (a *builtinApp) list(r *incoming.Request, creq apps.CallRequest) apps.CallR
 	includePluginApps := creq.BoolValue("plugin-apps")
 
 	listed := a.proxy.GetListedApps(r, "", includePluginApps)
-	installed := a.proxy.GetInstalledApps(r)
+	installed, reachable := a.proxy.GetInstalledApps(r, true)
```

Comment:
From https://github.com/mattermost/mattermost-plugin-apps/pull/325, will disappear from the diffs soon?